resources/assets/less/bem/profile-extra-account-standing.less

```diff
@@ -0,0 +1,134 @@
+/**
+ *    Copyright 2015-2018 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.profile-extra-account-standing {
+  @_top: profile-extra-account-standing;
+
+  &__alert {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    width: 100%;
+    height: 30px;
+    border-radius: 15px;
+
+    margin-bottom: 5px;
+    padding-left: 20px;
+
+    &--info {
+      background-color: #ccc;
+    }
+
+    &--warning {
+      background-color: @red-dark;
+      color: #fff;
+    }
+  }
+
+  &__action {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    height: 20px;
```

Comment:
Add 2px padding top/bottom instead (and remove the flex display). Make border-radius something like 2*side-padding so it stays stadium-shaped. Also for `__alert`.